lib/main.js

```diff
@@ -6,7 +6,7 @@ import logger from './logger'; // logger needs to remain first of imports
 import _ from 'lodash';
 import { server as baseServer } from 'appium-base-driver';
 import { asyncify } from 'asyncbox';
-import getParser from './parser';
+import getParser, { getDefaultArgs } from './parser';
```

Comment:
This works?

I've always used `import { default as getParser, getDefaultArgs } from './parser';`